executor/grant.go

```diff
@@ -473,6 +473,13 @@ func (e *GrantExec) grantDBLevel(priv *ast.PrivElem, user *ast.UserSpec, interna
 	if priv.Priv == mysql.UsagePriv {
 		return nil
 	}
+	globalOnlyPrivs := mysql.Privileges{mysql.ProcessPriv, mysql.ShowDBPriv, mysql.SuperPriv, mysql.CreateUserPriv, mysql.CreateTablespacePriv, mysql.ShutdownPriv, mysql.ReloadPriv, mysql.FilePriv, mysql.ReplicationClientPriv, mysql.ReplicationSlavePriv}
```

Comment:
Could we unexplicit define globalOnlyPrivs? Use AllGlobalPrivs and AllDBPrivs to get the result?

Reply:
I had tried AllGlobalPrivs and AllDBPrivs，but it's not match.  globalOnlyPrivs comes from mysql document and my practice.